# Final PCL Data message typedef
float32[4] height
float32[4] width
float32[4] ratio
geometry_msgs/Point[4] centroid